demo/java/src/main/java/io/github/cosinekitty/astronomy/demo/Main.java

```diff
@@ -66,8 +54,7 @@ public static void main(String[] args) {
                     break;
 
                 case "now":
-                    Date now = new Date();
-                    Time time = Time.fromMillisecondsSince1970(now.getTime());
+                    Time time = Time.fromMillisecondsSince1970(System.currentTimeMillis());
```

Comment:
`System.currentTimeMillis` is another naming for the same concept but is way older naming. It can be used instead `new Date().getTime()` so I used that instead.